test/e2e-test/nextjs.test.ts

```diff
@@ -0,0 +1,120 @@
+        dataToArray.splice(61,1,'               <img src="/vercel.svg" alt="Vercel Logo" width="72" height="16" />\n');
+        const arrayToText =  dataToArray.join('\n');
+        fs.writeFileSync(`${template}/pages/index.js`,arrayToText)
+        process.chdir(template);
+        await execFile(`npx next build`);
+        await execFile(`npx next export`);
+    });
+
+    it('Clone cell site template', async () => {
+        process.chdir(template);
+        const { stdout} = await execFile(`azion-framework-adapter init cells-site-template https://github.com/aziontech/cells-site-template`);
+        expect(stdout).to.be.equal('Completed.\n');
+    });
+
+    it('Install dependencies', async () => {
+        process.chdir('./cells-site-template');
+        await execFile('npm install');
+        const packageLock = fs.existsSync(path.join(template,'./cells-site-template/package-lock.json'));
+        expect(packageLock).to.be.true;
+    });
+
+
+    it('Copy azion.json file ', async () => {
+        await copy(path.join(localOutput, 'test', 'config-files', 'azion-next.json'),path.join(template,'./cells-site-template/azion.json'))
+        const azionConfigFile = fs.existsSync(path.join(template,'./cells-site-template/azion.json'));
+        expect(azionConfigFile).to.be.true;
+    });
+
+    it('Build the nextjs project', async () => {
+        const expectOutput = `Wrote manifest file to ${template}/cells-site-template/worker/manifest.json\n`+
+        `Finished worker.\n`+
+        `Completed.\n`
+        const { stdout } = await execFile('azion-framework-adapter build --static-site --assets-dir ../out');
+        const nextjsOutputDir = fs.existsSync(path.join(template,'/cells-site-template/worker/function.js'));
+        expect(stdout).to.be.equal(expectOutput);
+        expect(nextjsOutputDir).to.be.true;
+    });
+
+    it('Publish only asset of the nextjs to S3', async () => {
+        const { stdout } =await execFile('azion-framework-adapter publish --only-assets --assets-dir ../out');
+        const publishOutputMessage = `Loading asset manifest from ${template}/cells-site-template/worker/manifest.json\n`;
+        expect(stdout).to.be.equal(publishOutputMessage);
+    });
+
+    it("Compare manifest with the list of files on S3 Bucket", async () => {
+        const filesOnS3Bucket: any[] = [];
+        async function getAllKeys(bucketParams: AWS.S3.ListObjectsV2Request,  filesOnS3Bucket: any[]){
+            const response = await s3.listObjectsV2(bucketParams).promise();
+            response.Contents?.forEach(obj => filesOnS3Bucket.push(obj.Key?.replace('__static_content_test/nextjs-e2e/', '')));
+
+            if (response.NextContinuationToken) {
+                bucketParams.ContinuationToken = response.NextContinuationToken;
+                await getAllKeys(bucketParams, filesOnS3Bucket);
+            }
+            return filesOnS3Bucket;
+        }
+
+        await getAllKeys(bucketParams, filesOnS3Bucket);
```

Comment:
It seems this test is relying on the previous test output. Probably both tests should be made independent or merged. To speedup  the process, I favor the merge.